src/main/java/com/artipie/SliceFromConfig.java

```diff
@@ -158,7 +158,7 @@ static Slice build(final Settings settings, final Authentication auth,
                 break;
             case "npm":
                 slice = new NpmSlice(
-                    cfg.path(),
+                    "/npm",
```

Comment:
@victornoel the path is not correct here, it's dynamic string take from URI path, it should be full repository name

Reply:
@g4s8 yes, that was my question in the issue and in the description of this MR. What is the "full repository name"? It is not clear to me

Reply:
@g4s8 can you clarify this please?

Reply:
@victornoel the full repository name is a string which may contains slash char as a separator. It's trailing part of request URI. It depends on layouts structure: we support two layouts `flat` and `org` - `flat` has only repo name of repository in a path, `org` has user name and repo name in a path. E.g. with `org` layout user `user` in repository `repo` access artifacts by URI `some-host.com/user/repo/artifact-id`, with `flat` layout it will be `some-host.com/repo/artifact-id`.

Reply:
@g4s8 I see there is now a `TrimPathSlice` object that wraps most of the `Slice` and that those slice do not take a path parameter or if they do, it is set to `""`.
Can you tell me if I should use that instead of passing the path (which I still don't know where to find even after your answer, sorry).

Reply:
@g4s8 I went with this, can you check it? :)